packages/graphql/tests/schema/directives/cypher.test.ts

```diff
@@ -600,7 +743,15 @@ describe("Cypher", () => {
             type Query {
               movies(options: MovieOptions, where: MovieWhere): [Movie!]!
               moviesAggregate(where: MovieWhere): MovieAggregateSelection!
-              moviesConnection(after: String, first: Int, where: MovieWhere): MoviesConnection!
+              moviesConnection(after: String, first: Int, sort: [MovieSort], where: MovieWhere): MoviesConnection!
```

Comment:
Same for this, I don't think it should be sortable?

Reply:
Why not? we have sort on cypher fields I believe, or there are other reason why it should not be sortable?

Reply:
Because the Cypher field in question here has a parameter, which we can't factor into sorting!

Reply:
Aha! good catch I forgot this case!